cardInfoAPI-Level1.yaml

```diff
@@ -0,0 +1,1402 @@
+        card_id:
+          $ref: '#/components/schemas/CardId'
+        related_transaction_id:
+          $ref: '#/components/schemas/RelatedTransactionId'
+
+    ContractReference:
+      title: Contract reference
+      type: string
+      maxLength: 50
+      examples:
+        - CH3456789012345678901
+        - ABC-56789012345
+      description: Functional reference to a customer relationship/contract that is responsible for the card, defined by the Card Service Provider.
+
+    PersonReference:
+      title: Person reference
+      type: string
+      maxLength: 50
+      examples:
+        - '123456789'
+      description: Functional reference to a cardholder, defined by the Card Service Provider.
+
+    CardId:
+      title: Card id
+      type: string
+      format: uuid
+      maxLength: 50
+      examples:
+        - 4321A1B2C3DE1234
+      description: Technical reference to a card (unique, constant in the lifecycle of a card even in case of replacements or renewals).
+
+    EmbossingLine1:
+      title: Embossing line 1
+      type: string
+      maxLength: 35
+      examples:
+        - Peter Schweizer
+      description: First line embossed on the card to indicate the cardholder.
+
+    EmbossingLine2:
+      title: Embossing line 2
+      type: string
+      maxLength: 35
+      examples:
+        - Peter Schweizer
+      description: Second line embossed on the card to indicate the cardholder.
+
+    Pan4Digits:
+      title: Pan 4 digits
+      type: string
+      maxLength: 4
+      pattern: '^\d{4}$'
+      examples:
+        - '4321'
+      description: Last 4 digits of the PAN; values can change for the same cardId during the lifecycle.
+
+    ExpiryDate:
+      title: Expiry date
+      $ref: '#/components/schemas/Date'
+      description: Last day when the card is valid and can be used for payments.
+
+    InitialIssuingDate:
+      title: Initial issuing date
+      $ref: '#/components/schemas/Date'
+      description: Date when card was issued initially.
+
+    CardStatus:
+      title: Card status
+      type: string
+      enum:
+        - ACTIVE
+        - SUSPENDED
+        - BLOCKED
+      examples:
+        - ACTIVE
+      description: |
+        Effective status of the card to indicate whether it can be used for payments; definition of enums:
+        - active: authorizations are possible
+        - suspended: authorizations are temporarily not possible
+        - blocked: authorizations are permanently not possible because the card was cancelled or permanently blocked for other reasons
+
+    Currency:
+      type: string
+      pattern: '^[a-zA-Z0-9]{3}$'
+      examples:
+        - CHF
+        - EUR
+      description: Settlement currency of the card / card account in ISO code format.
+
+    Image:
+      type: string
+      format: uri
+      examples:
+        - https://one-digitalservice.ch/public/Viseca/media/one-app-images/card-visuals/4_go_m_11_2017.png
```

Comment:
How are these images being accessible? Public no apikey? Or do we need to provide an API for images?

Reply:
Our idea is that the URI is publicly accessible.

Reply:
Where are the images for the cards stored? Are they always by the PSP, by issuer or third parties?

Reply:
Images are available from the data provider.